acceptance/suites/tests/https/client_may_use_external_cert_chains.rb

```diff
@@ -1,5 +1,8 @@
 test_name "Ensure Puppet Server's HTTP client may use external cert chains" do
 
+  ## This test currently relies on WEBrick/OpenSSL, which is not supported on FIPS
```

Comment:
That's weird, we run the webrick server using the agent's Ruby which I thought has OpenSSL available to it. Was there anything in the `webrick.log`?

Reply:
Ah sorry @mwaggett I think I gave you bad advice earlier. I didn't realize the first part of the test used webrick and the second used puppetserver's ruby. To @justinstoller point, the first part should work on FIPS hosts since we're using the agent's ruby to load net/http and openssl.

Reply:
ahh @justinstoller @joshcooper so you're saying this *should* work on FIPS? Should we revert this commit then? https://github.com/puppetlabs/puppetserver/pull/2645/commits/6e281516fdffb430ad91c9a4a886d8bff2d24231

Reply:
I think @jonathannewman merged it to unblock CI and let folks investigate the FIPS failures at a lower priority.

Reply:
@jonathannewman Is there a ticket to investigate the FIPS failures and revert https://github.com/puppetlabs/puppetserver/commit/6e281516fdffb430ad91c9a4a886d8bff2d24231 once we get it figured out?

Reply:
I have not written one.  Feel free to.  As I understood it, these tests weren't expected to pass in this environment.

Reply:
That's what I had originally thought, but it sounds like @justinstoller and @joshcooper are saying it actually _should_ work on FIPS, since we use the agent's Ruby. Am I understanding that right?

Reply:
There's issues in general with HTTPS connections in Server on FIPS, so I don't know how important it is that we can run this test on FIPS right now.

However, the stated reason for the failure in this comment shouldn't be an issue on FIPS because we're not loading Webrick in JRuby.

Reply:
@justinstoller I'm trying to determine if there's a need to open a followup ticket to re-enable this test on FIPS, since it sounds like this _should_ work. I don't know about these "issues in general with HTTPS connections in Server on FIPS" - is that something we expect to resolve at some point? is there a specific ticket (or tickets) for fixing this? or should we never plan to run this test on FIPS..?

Reply:
I think you should create a ticket. I was referring to the fact Net::HTTPS / OpenSSL library doesn't work in JRuby on FIPS.

Reply:
okay, opened https://tickets.puppetlabs.com/browse/PE-34102 to resolve the failure and https://github.com/puppetlabs/puppetserver/pull/2647 to fix the comment.